eng/Version.Details.xml

```diff
@@ -11,6 +11,15 @@
       <Sha>01850f2b7bff23e118d1faecb941d770c8e626f2</Sha>
       <SourceBuild RepoName="source-build-reference-packages" ManagedOnly="true" />
     </Dependency>
+    <Dependency Name="System.CommandLine" Version="2.0.0-beta4.23307.1">
```

Comment:
Does we need to ensure this version always matches the version in `Versions.props` - https://github.com/dotnet/roslyn/blob/main/eng/Versions.props#L234 ?

Reply:
dependency flow takes care of that. Those versions aren't updated manually... usually.